Bottom: 78f973e1979685e846736a60c1af42d5184c85c6
Top:    63a13b806527e5589d3dca0330e50467eb98d5b6
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-10-27 16:53:30 +0000

find.c: The User varaible needs to die...

Reduce its usage to where absolutely necessary... oh, what a surprise,
another variable passing state down into add_object_to_list ()


---

diff --git a/src/find.c b/src/find.c
index af5aa3e..70cc17f 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3090,7 +3090,7 @@ LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int flag,
    */
   ListStart (type, ptr1, ptr2, ptr3, flag);
   DoIt (flag, AndRats, AndDraw);
-  if (User)
+  if (AndDraw)
     IncrementUndoSerialNumber ();
   User = false;
